zebra-consensus/src/transaction/tests.rs

```diff
@@ -241,3 +241,45 @@ async fn v5_transaction_is_accepted_after_nu5_activation() {
         assert_eq!(result, Ok(expected_hash));
     }
 }
+
+#[tokio::test]
+// TODO: Remove `should_panic` once the NU5 activation heights for testnet and mainnet have been
+// defined.
+#[should_panic]
```

Comment:
I'm a bit worried that the activation height is set but the assertion fails, which could lead us to forget to remove this `#[should_panic]`. I tested with a fake activation height, and it seems to work, but I'm still not 100% sure.

Reply:
Feel free to add a list of these tests to the activation height ticket #1841, with links to the code for each test

Reply:
Thanks! That will help to keep track of these tests :smile: